static/app/components/feedback/feedbackItem/screenshotSection.tsx

```diff
@@ -0,0 +1,114 @@
+import {useState} from 'react';
+
+import {
+  useDeleteEventAttachmentOptimistic,
+  useFetchEventAttachments,
+} from 'sentry/actionCreators/events';
+import {openModal} from 'sentry/actionCreators/modal';
+import Section from 'sentry/components/feedback/feedbackItem/feedbackItemSection';
+import {t} from 'sentry/locale';
+import type {Organization} from 'sentry/types';
+import type {Event} from 'sentry/types/event';
+import type {EventAttachment} from 'sentry/types/group';
+import {objectIsEmpty} from 'sentry/utils';
+
+import Screenshot from '../../events/eventTagsAndScreenshot/screenshot';
+import Modal, {modalCss} from '../../events/eventTagsAndScreenshot/screenshot/modal';
```

Comment:
l: let's use the absolute path as it's a bit easier to read (otherwise you have to know/lookup what the current file path is)